### PR TITLE
Tts websockets example

### PR DESCRIPTION
**Pull request by mediumTaj**
### Summary
This PR adds an example for websockets

### CODE REVIEW

**Review by kevinkowa — CHANGES_REQUESTED** [review]
do we need this file?

File: IBM.Watson.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2036
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31112.23
 MinimumVisualStudioVersion = 10.0.40219.1
```
Comment (kevinkowa) [inline IBM.Watson.sln:5]: do we need this file?

**Review by kevinkowa — CHANGES_REQUESTED** [review]

File: IBM.Watson.sln (hunk 1 above)
Comment (mediumTaj) [reply IBM.Watson.sln:5]: Yes, its the main solution for all of the projects. I updated my visual studio so the version updated. I dont think its a big deal.

**Comment by mediumTaj** [comment]
hmm i am getting an error in CI complaining about .net sdk not supporting .net 3.x. this might be an issue.

**Comment by kevinkowa** [comment]
hmm odd, is there a way to skip the CI for that example folder?

**Comment by mediumTaj** [comment]
I don’t think so. Maybe we should remove the playing example and just show the streaming.

**Review by kevinkowa — APPROVED** [review]
LGTM

**Pull request merged**